src/main/java/com/dumbdogdiner/stickyapi/bukkit/util/PlayerUtil.java

```diff
@@ -0,0 +1,126 @@
+/**
+ * Copyright (c) 2020 DumbDogDiner <dumbdogdiner.com>. All rights reserved.
+ * Licensed under the MIT license, see LICENSE for more information...
+ */
+package com.dumbdogdiner.stickyapi.bukkit.util;
+
+import org.bukkit.Bukkit;
+import org.bukkit.OfflinePlayer;
+import org.bukkit.entity.Player;
+
+import java.util.*;
+
+import static com.dumbdogdiner.stickyapi.common.util.NumberUtil.longToInt;
+
+
+//FIXME we need to have a login logout listener, and tbh this should be done by a DB, and this class should act as a basic db interface or similar. Probably need a combined waterfall-bukkit plugin for this, or maybe just waterfall. I honestly do not know.
+@SuppressWarnings("unused")
+public class PlayerUtil {
+    public static class Comparators {
+        public static final Comparator<OfflinePlayer> BY_LAST_SEEN = new OfflinePlayerSeenComparator();
```

Comment:
There's probably a way you can do this using lambdas.

Reply:
^